config/detekt/detekt-config.yml

```diff
@@ -47,6 +47,18 @@ potential-bugs:
     active: true
 
 style:
+  excludes:
+    - "**/resources/**"
+    - "**/build/**"
+    - '**/test/**'
+    - '**Test.kt'
```

Comment:
Not sure if we want to exclude test files. What do others think?

Reply:
This should be the equivalent of https://github.com/allegro/envoy-control/blob/master/config/detekt/default-detekt-config.yml which isn't supported anymore. It needs a major version upgrade anyway - maybe I'll should clean it up in a next PR?

Reply:
OK, so it is also not a change compared to current rules. We are not in hurry to upgrade detekt, but if you want to do it that's great.

Reply:
Added new PR https://github.com/allegro/envoy-control/pull/176 to be considered after this one is merged, I've adapted the config there to only exclude tests on problematic rules and not for the whole set of them.